project-reports/2023/2023-Q4-Hyperledger-Caliper.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: 2023 Q4 Hyperledger Caliper
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+* 2 active maintainers
+* Questions on Discord and GitHub are regularly answered.
+* Monthly contributor calls if needed (discussion happens mostly on Discord).
+* No project core contributions.
+
+# Questions/Issues for the TOC
+
+Not at this time.
+
+# Releases
+
+The latest stable release is v0.5.0; we're also continuously publishing unstable releases after every merged PR (no new releases):
+
+* [npm](https://www.npmjs.com/package/@hyperledger/caliper-cli/v/0.5.0)
+* [DockerHub](https://hub.docker.com/layers/caliper/hyperledger/caliper/0.5.0/images/sha256-3a7df18ef6a17a8c851dac19241049be2375dc892643c576932f360ca9d7198c?context=explore)
+
+# Overall Activity in the Past Quarter
+
+The activity has been minimal in the last quarter due to the maintainers being otherwise engaged. There's an ongoing discussion about establishing a performance-related group (of some form), along with preparing workshop materials for Webinars in 2024. We think such publicity will do good for Caliper in terms of gaining more contributors.
+
+# Current Plans
+
+There are no current plans, I would expect that the main interested parties of Hyperledger Fabric and Hyperledger Besu would plan to contribute specifically to the Caliper connectors in order to ensure that Caliper continues to meet the needs of a load generator for them. However if within 6 months starting 1 Jan 2024, Caliper doesn't get any active contributors and thus see some activity then we should consider moving Caliper to the "dormant" state of the Hyperledger project lifecycle.
```

Comment:
Hyperledger Cacti also has performance benchmarking needs, and we should discuss whether Caliper is the appropriate place for measuring and benchmarking transactions and applications (load generation, TPS and latency measurements, report generation) that span multiple networks. But even if the answer to this question is yes, I'm not sure Cacti maintainers will have the bandwidth to contribute to Caliper within the next 6 months.